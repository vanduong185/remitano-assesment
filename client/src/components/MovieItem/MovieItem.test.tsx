import React from "react";
import { render } from "@testing-library/react";
import MovieItem from ".";

it("Should match snapshot", async () => {
  const { container } = render(
    <MovieItem
      movie={{
        id: 1,
        movieDescription: "description",
        movieTitle: "title",
        movieUrl: "https://www.youtube.com/watch?v=B8p1NB9eYcM",
        user: { username: "username" },
        userId: 1,
      }}
    />
  );
  expect(container).toMatchSnapshot();
});
