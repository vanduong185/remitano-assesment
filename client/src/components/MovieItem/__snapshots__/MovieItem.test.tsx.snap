// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container css-1423t8p-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
    >
      <div
        class="css-1fumm8e"
      >
        <iframe
          class="css-122y91a"
          frameborder="0"
          src="https://www.youtube.com/watch?v=B8p1NB9eYcM"
        />
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
    >
      <h5
        class="MuiTypography-root MuiTypography-h5 css-1jcm1rz-MuiTypography-root"
      >
        title
      </h5>
      <p
        class="MuiTypography-root MuiTypography-body1 css-z2eky3-MuiTypography-root"
      >
        Shared by: 
        username
      </p>
      <p
        class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
      >
        Description: 
        description
      </p>
    </div>
  </div>
</div>
`;
